harness/determined/pytorch/_trainer.py

```diff
@@ -41,6 +40,8 @@ def configure_profiler(
         end_after_batch: Optional[int] = None,
     ) -> None:
         """
+        @deprecated: Configure `fit(..., profiling_enabled=True) instead`.
```

Comment:
Since this removes `profiler.ProfilerAgent`, what do you think about making this an error rather than a warning? Because the two profilers are different, I think it's a big deal that people expecting the old profiler need to know about.

Another suggestion is to just remove the method entirely.

Or to provide a link to docs. This is more than just a change in interface, but also a change to functionality.

Reply:
changed this to an error.

Reply:
nice